contrib/mesos/docs/issues.md

```diff
@@ -1,34 +1,120 @@
-## Known Issues
+## DCOS Package Known Issues
```

Comment:
It's super confusing have this as the page title when not all of the issues are DCOS specific.

It'd be ideal to just have a "Known Issues" page title, folowed bya  table of contents that shows that there are general issues followed by dcos-specific issues. Thent he DCOS section can say that all of the general issues apply, and these additional issues also apply.